pkg/util/ovs/fake_ovs.go

```diff
@@ -161,12 +163,12 @@ func (fake *ovsFake) Clear(table, record string, columns ...string) error {
 }
 
 type ovsFakeTx struct {
-	fake *ovsFake
-	err  error
+	fake  *ovsFake
+	flows []string
 }
 
 func (fake *ovsFake) NewTransaction() Transaction {
-	return &ovsFakeTx{fake: fake, err: fake.ensureExists()}
+	return &ovsFakeTx{fake: fake, flows: []string{}}
```

Comment:
Note that there's a mild behavior change here: it used to check that you had called `ovsif.AddBridge()` on the `ovsFake`, but now it doesn't. (We apparently don't test this case in `fake_ovs_test.go`.) Anyway, you could just move the `fake.ensureExists()` call to Commit